packages/core/src/sheets/workbook.ts

```diff
@@ -87,7 +87,7 @@ export class Workbook extends Disposable {
     ) {
         super();
 
-        this._snapshot = Tools.commonExtend(DEFAULT_WORKBOOK, workbookData);
+        this._snapshot = Tools.deepClone(Tools.commonExtend(DEFAULT_WORKBOOK, workbookData));
```

Comment:
Either perform a deep clone of `DEFAULT_WORKBOOK` or create a `getEmptySnapshot` function, similar to what is done in `packages/core/src/docs/data-model/empty-snapshot.ts`.

Reply:
Got it and have solved it.